.github/workflows/release-notice.yml

```diff
@@ -19,5 +19,8 @@ jobs:
     - name: Notify Slack
       id: slack
       with:
-        slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_RELEASES_URL }}
-      uses: fedecalendino/slack-release-notifier@8329c1c2d20055ff874ce3eaed7021735d17069d # v2.1.0
+        project_name: "OpenImageIO"
+        slack_bot_token: ${{ secrets.SLACK_BOT_TOKEN }}
+        slack_channel: "#release-announcements"
+        project_logo: "https://artwork.aswf.io/projects/openimageio/stacked/color/openimageio-stacked-color.png"
+      uses: jmertic/slack-release-notifier@803345f
```

Comment:
You might want to do `@main`, only because I'm sure we will find bunches of bugs and this way you will get the latest fixes.

Reply:
Can you do this, since you need to adjust your DCO anyway?